docker-entrypoint.sh

```diff
@@ -2,11 +2,18 @@
 set -e
 
 mkdir -p $TEAMCITY_DATA_PATH/lib/jdbc $TEAMCITY_DATA_PATH/config
-if [ ! -f "$TEAMCITY_DATA_PATH/lib/jdbc/postgresql-9.3-1103.jdbc41.jar" ];
+
+if [ "$DATABASE" == "mysql" ];
+then
+    if [ ! -f "$TEAMCITY_DATA_PATH"/lib/jdbc/mysql-connector-java.jar ];
```

Comment:
I would rather use the same appoach as for the postgress jar, e.g. download it from http://central.maven.org/maven2/mysql/mysql-connector-java/5.1.38/mysql-connector-java-5.1.38.jar for example instead of using system package, this also eliminates the changes in the Dockerfile